src/GitTools.Core/GitTools.Core.Shared/Git/Helpers/GitRepositoryHelper.cs

```diff
@@ -12,7 +12,7 @@
     /// </summary>
     public static class GitRepositoryHelper
     {
-        static readonly ILog Log = LogProvider.GetCurrentClassLogger();
+        static readonly ILog Log = LogProvider.GetLogger(typeof(GitRepositoryHelper));

```

Comment:
GetCurrentClassLogger isn't available in portable LibLog, I think this is equivalent.